### DISCUSSION TITLE
Support for babel-plugin-transform-object-rest-spread

### DISCUSSION THREAD

**Discussion by noalwix** [swc-project/swc#1398] [Q&A] [answered]
Hi all!

I couldn't find in the [Comparison with babel page](https://swc.rs/docs/comparison-babel/) about babel-plugin-transform-object-rest-spread. Is it supported?

Thanks!

**Comment by kdy1** [comment, marked as answer]
Yes, all syntax in official ecmascript spec is supported. I'll update the document.

**Reply by noalwix** [reply]
Thanks for the quick response!

**Reply by noalwix** [reply]
To make sure - this is true for `babel-preset-es2015-node6` too right?

**Reply by kdy1** [reply]
Sorry but I'm not sure because I don't know well about babel preset.
